webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineEdit.stories.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import Story from '../../../../../../stories/components/Story';
+import InlineEdit from './InlineEdit';
+
+export default {
+  title: 'Components|InlineEdit',
+};
+
+// eslint complains about hooks in storybooks
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
After removing this I didn't see any console warnings, and `npm run lint` completed with no errors.  I'd say it's safe to remove this